Update persons.html

Chigoose opened the pull request.

Chigoose committed:
Update persons.html

alexey0097 commented:
1. **Нужно изменить название заголовка**
В заголовке на Pull Request (PR) принято делать наименование задачи.
_Например: "Задача 1. Обновление раздела с персоналиями"_

2. **В блоке с комментарием надо коротко описать что ты сделал**
Можно добавить полезную информацию для других разработчиков, которая поможет им разобраться в твоем коде.

3. **Не забывай постоянно обновлять проект**
В нем могут быть изменения, внесенные другими разработчиками. Я обнаружил что ты проглядел один блок кода. Это поможет избежать Merge Conflicts.
_Роберт Мартин в своей книге "Clean Architecture" называет это "синдром вчерашнего дня". Когда ты заканчиваешь работать в пятницу вечером, любой другой разработчик может изменить твой код в субботу и воскресенье. В понедельник утром тебя будет ждать неприятный сюрприз._

alexey0097 commented on persons.html:
```diff
@@ -278,18 +280,26 @@ <h5 class="modal-title" id="youtubeModalLabel">Обращение к
 
         <p><big>Структура местных (первичных) отделений</big></p>
 
-        <table>
-            <tr class="chapter-row">
-                <th class="chapter-cell">МО в г. Чебоксары</th>
-                <th class="chapter-cell">Милюхин Кирилл Владимирович</th>
-            </tr>
-            <tr>
-                <td>ПО в г. Новочебоксарске</td>
-                <td>Фирсов Дмитрий Николаевич</td>
-            </tr>
-        </table>
```
**Вот этот блок кода удалять не нужно**

Скорее всего это связанно с тем, что ты не обновил проект и использовал его старую версию. Не забывай обновляться - это важно.

Для обновления проекта используется команда `git pull`. Ссылка на документацию к этой команде [тут](https://git-scm.com/docs/git-pull).

Тебе нужно заменить Ф.И.О. не удаляя этого блока.

alexey0097 commented on persons.html:
```diff
@@ -278,18 +280,26 @@ <h5 class="modal-title" id="youtubeModalLabel">Обращение к
-            </tr>
-        </table>
+        <div class="container-two">
+<div>
+<p class="names">МО в г. Чебоксары</p>
+</div>
+<div>
+<p class="names">Милюхин Кирилл Владимирович</p>
+</div>
+</div>
+
 
-    </div>
+<div class="container-three"> 
+<div>
+<p class="names-2">ПО в г. Новочебоксарске</p>
+</div>
+<div>
+<p class="names-3">Фирсов Дмитрий Николаевич</p>
+</div>
+</div>
+
+</div>
```
**Не забывай про выравнивание в коде**

Другие разработчики явно не обрадуются когда увидят это. Поверь мне, знаю не по наслышке.

Подобные блоки кода лучше выравнивать `TAB`. Если не получается сделать красивый отступ - используй пробелы.

>  Программирование это не только профессия, а стиль жизни и образ мышления. Порядок в жизни начинается с порядка в коде.

Chigoose committed:
Update persons.html (Задача 1: Обновление раздела с персоналиями)

Добавил в раздел с персоналиями (блок <div class="container col-lg-12 col-xl-10 col-xxl-8"><div>) несколько новых имен. В список "Бюро регионального совета" добавил имена " Васильева Елена Николаевна" и "Карпушин Дмитрий Вячеславович". Список же " Контрольно-ревизионная комиссия" полностью обновил, заменив имена Васильева Елена Николаевна, Петров Александр Геннадьевич и Семенов Геннадий Германович" на "Горохов Александр Викторович, Фирсов Дмитрий Николаевич и Янгаров Алексей Николаевич".

Chigoose committed:
Update persons.html
